Realm/RLMSyncCredentials.h

```diff
@@ -89,6 +95,16 @@ extern RLMIdentityProvider const RLMIdentityProviderJWT;
  */
 + (instancetype)credentialsWithJWT:(NSString *)token;
 
+/**
+ Construct and return anonymous credentials
+ */
++ (instancetype)credentialsWithAnonymous;
```

Comment:
`credentialsWithAnonymous` doesn't make much sense. The `WithFoo` naming is used to label an argument. `anonymousCredentials` would be a better choice.